packages/stargate/src/modules/authz/queries.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { makeCosmoshubPath } from "@cosmjs/amino";
+import { coins, DirectSecp256k1HdWallet } from "@cosmjs/proto-signing";
+import { Tendermint34Client } from "@cosmjs/tendermint-rpc";
+import { sleep } from "@cosmjs/utils";
+import { GenericAuthorization } from "cosmjs-types/cosmos/authz/v1beta1/authz";
+
+import { QueryClient } from "../../queryclient";
+import { SigningStargateClient } from "../../signingstargateclient";
+import { assertIsDeliverTxSuccess } from "../../stargateclient";
+import {
+  defaultSigningClientOptions,
+  faucet,
+  pendingWithoutSimapp44,
+  simapp,
+  simapp44Enabled,
+} from "../../testutils.spec";
+import { AuthzExtension, setupAuthzExtension } from "./queries";
+
+async function makeClientWithAuthz(
+  rpcUrl: string,
+): Promise<[QueryClient & AuthzExtension, Tendermint34Client]> {
+  const tmClient = await Tendermint34Client.connect(rpcUrl);
+  return [QueryClient.withExtensions(tmClient, setupAuthzExtension), tmClient];
+}
+
+describe("AuthzExtension", () => {
+  const defaultFee = {
+    amount: coins(25000, "ucosm"),
+    gas: "1500000", // 1.5 million
+  };
+  const granter1Address = faucet.address1;
+  const grantee1Address = faucet.address2;
```

Comment:
Can we use a random address for this one which does not exist (see `makeRandomAddress` from `packages/cosmwasm-stargate/src/testutils.spec.ts`)? Then we get a new (granter, grantee) pair each time we run the test and exactly one expected result element.